_capabilities/utdallas-ufl-xar-demo.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Explainable Video Activity Recognition Demo"
+excerpt: "Demo of our XAI system built to detect activities in videos with post-hoc explanations for its predictions."
+tags: 
+- Analytics
+- Computer Vision
+- Visual Question Answering (VQA)
+- Human-Machine Teaming
+- Explanation Framework
+
+submission_details:
+resources: 
+papers: 
+- title: Don’t Explain without Verifying Veracity: An Evaluation of Explainable AI with Video Activity Recognition
+url: https://arxiv.org/abs/2005.02335
+software:
+- title: Explainable Video Activity Recognition Demo
+url: https://github.com/MahsanNourani/XAI-Video-Explanation
+
+
+
+authors:
+- Mahsan Nourani <sup>2</sup>
+- Chiradeep Roy <sup>1</sup>
+- Tahrima Rahman <sup>1</sup>
+- Eric Ragan <sup>2</sup>
+- Nicholas Ruozzi <sup>1</sup>
+- Vibhav Gogate  <sup>1</sup>
+organizations:
+- 1. The University of Texas at Dallas
+- 2. University of Florida
+point_of_contact:
+name: Eric Ragan
+email: eragan@ufl.edu
+---
+
+## Overview
+Machine Learning Systems are often times referred to as black-boxes, as their users cannot properly understand how to use them. Explainable Artificial Intelligence (XAI) systems are therefore used to bring transparency for the end-users to help them understand such systems and build a better mental model of these systems. Through this project, we built an explainable tool for activity recognition in cooking videos, where the system provides yes/no responses to whether an activity is happening in a video, and the system's justification for that output. We then used this system to explore how presence and quality of explanations can affect certain user behaviors, such as user-task performance, trust, and understanding. 
+
+
+## Intended Use
+The tool illustrates how to build a user-friendly visual interface and the different explanation types that one should provide to a user for solving the task of explainable activity recognition in video data. At a high level, the system works as follows. The user uploads a video and asks a yes/no question (e.g., did the user perform activity X) about the video. The system processes the video, answers the question and provides three types of explanations, explaining the yes or no answer to the question.
+
```

Comment:
Given the demo probably runs on data (looks like the TACoS dataset from the paper), I think it would be useful to mention that here in the "Model/Data" section with potentially a pointer to the actual dataset. Note that we also have a data field in the header section which can be used to link to datasets.